### ISSUE TITLE
'Clear and download current view' menu item color hints

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#2009]
Regarding these two menu items next to each other:

- Download current view
- Clear and download current view

Perhaps the latter could be in green color, when if one presses it there will be no warning produced. 

In other words, when there is no pending changes, and we might as well clear first, especially if we are now in a new area.

And it could be in a red color, if pressing it will cause us to regret pressing it. E.g., when there are pending changes that haven't been uploaded yet.

So in summary, with some color clues, we could have advanced information about if, and if not, we might press it.

**Comment by simonpoole** [comment]
This doesn't make sense, it would just duplicate the pending changes display.
